sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select *
+from "customers"
+where "isActive" = 'false';
```

Comment:
isActive is a boolean column, so just use `true` or `false` without quotes.
`"isActive" = false`